projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -956,6 +957,41 @@ function ( $site_purchase ) {
 	return empty( $bundle_purchases );
 }
 
+/**
+ * Determines whether or not the Install the mobile app task should be visible.
+ *
+ * @return bool True if the Install the mobile app task should be visible.
+ */
+function wpcom_launchpad_is_mobile_app_installed_visible() {
+	$is_atomic_site = ( new Automattic\Jetpack\Status\Host() )->is_woa_site();
+	// If the site is not an Atomic site, we should not show the task.
+	if ( $is_atomic_site ) {
+		return false;
+	}
+
+	return true;
+}
+
+/**
+ * Verifies if the Mobile App is installed for the current user.
+ *
+ * @return bool True if the Mobile App is installed for the current user.
+ */
+function wpcom_launchpad_is_mobile_app_installed() {
+	if ( wpcom_is_checklist_task_complete( 'mobile_app_installed' ) ) {
+		return true;
+	}
+
+	$user_id          = get_current_user_id();
+	$mobile_last_seen = get_user_attribute( $user_id, 'jp_mobile_app_last_seen' );
```

Comment:
I don't think this will work on Atomic sites. I think we should add some defensive code to return early on Atomic sites, and then add some more defensive code to check that the function exists before we call it.
```suggestion
	if ( defined( 'IS_ATOMIC' ) && IS_ATOMIC ) { // I can't recall the constant to check
		return false;
	}

	if ( ! function_exists( 'get_user_attribute' ) ) {
		return false;
	}

	$mobile_last_seen = get_user_attribute( $user_id, 'jp_mobile_app_last_seen' );
```

Reply:
Good catch! Thanks!